risc0/zkvm/src/lib.rs

```diff
@@ -49,13 +47,13 @@ pub use risc0_zkvm_platform::{declare_syscall, memory::MEM_SIZE, PAGE_SIZE};
 
 #[cfg(feature = "binfmt")]
 pub use self::binfmt::{elf::Program, image::MemoryImage};
+pub use self::receipt::{SegmentReceipt, SessionReceipt};
 #[cfg(feature = "prove")]
-pub use self::exec::{Executor, ExecutorEnv, ExecutorEnvBuilder};
-#[cfg(feature = "prove")]
-pub use self::prove::{loader::Loader, Prover, ProverOpts};
-pub use self::receipt::{Receipt, SegmentReceipt, SessionReceipt};
-#[cfg(feature = "prove")]
-pub use self::session::{ExitCode, Segment, Session};
+pub use self::{
```

Comment:
Should we be Exposing `Syscall` and `SyscallContext` here? I might have missed if they are exposed to the public API somewhere else.

Reply:
We'll definitely follow up with this in a future PR, this one is already getting pretty big :)

Reply:
Sounds good to me.